arm/Microsoft.ContainerService/managedClusters/deploy.bicep

```diff
@@ -421,7 +529,13 @@ output resourceGroupName string = resourceGroup().name
 output name string = managedCluster.name
 
 @description('The control plane FQDN of the managed cluster')
-output controlPlaneFQDN string = (aksClusterEnablePrivateCluster ? managedCluster.properties.privateFQDN : managedCluster.properties.fqdn)
+output controlPlaneFQDN string = (enablePrivateCluster ? managedCluster.properties.privateFQDN : managedCluster.properties.fqdn)
 
 @description('The principal ID of the system assigned identity.')
 output systemAssignedPrincipalId string = systemAssignedIdentity && contains(managedCluster.identity, 'principalId') ? managedCluster.identity.principalId : ''
+
+@description('The Object ID of the AKS identity.')
+output kubeletidentityObjectId string = contains(managedCluster.properties, 'identityProfile') ? contains(managedCluster.properties.identityProfile, 'kubeletidentity') ? contains(managedCluster.properties.identityProfile.kubeletidentity, 'objectId') ? managedCluster.properties.identityProfile.kubeletidentity.objectId : '' : '' : ''
```

Comment:
Is the last check really necessary? -> Can there be a kubelet identity without an object Id?

Reply:
removed